client/src/pages/Main/Main.js

```diff
@@ -0,0 +1,178 @@
+      url: articleData["0"].web_url
+    })
+    .then(res => console.log("saved", articleData))
+    .catch(err => console.log(err));
+  };
+
+  handleFormSubmitNYTCall = event => {
+    event.preventDefault();
+    API.searchNYT({
+      searchTerm: this.state.searchTerm,
+      startDate: this.state.startDate,
+      endDate: this.state.endDate
+    })
+      .then(res => {
+        // console.log('this is response', response.data.response.docs);
+        this.setState({ 
+          articles: res.data.response.docs 
+        });
+      })
+      .catch(err => console.log(err));
+  };
+
+
+  render() {
+    return (
+      <Container fluid>
+        <Row>
+          <Col size="md-12">
+            <Jumbotron>
+              <h1>What NYT article would you like to look up?</h1>
+            </Jumbotron>
+            <form>
+              <Input
+                value={this.state.searchTerm}
+                onChange={this.handleInputChange}
+                name="searchTerm"
+                placeholder="Search Here (required)"
+                type="text"
+              />
+              <Input
+                value={this.state.startDate}
+                onChange={this.handleInputChange}
+                name="startDate"
+                placeholder="Starting Date"
+                type="number"
+              />
+              <Input
+                value={this.state.endDate}
+                onChange={this.handleInputChange}
+                name="endDate"
+                placeholder="Ending Date"
+                type="number"
+              />
+              <Form
+                disabled={!(this.state.searchTerm)}
+                onClick={this.handleFormSubmitNYTCall}
+              >
+                Find NYT Articles
+              </Form>
+            </form>
+          </Col>
+        </Row>
+
+
+
+        <Row>  
+          <Col size="md-12 sm-12">
+            <Jumbotron>
+              <h1>NYT Articles</h1>
+            </Jumbotron>
+            <Container>
+              {this.state.articles.length ? (
+              <List>
+
+
+              {this.state.articles.map(newsArticle => (
+                <ListItem
+                  key={newsArticle._id}
+                  id={newsArticle._id}
+                  title={newsArticle.headline.main}
+                  href={newsArticle.web_url}
+                  date={newsArticle.pub_date}
+                />
+                
+                <button disabled={!(this.state.searchTerm)} onClick={this.saveArticle}>
+                  save the article
+                </button>  
+
+              ))}
+              </List>
+              ) : (
+                <h3>Nothing to show yet</h3>
+              )}
+            </Container>
+          </Col>
+        </Row>  
+
+
+
+        <Row>  
+          <Col size="md-12 sm-12">
+            <Jumbotron>
+              <h1>Saved Articles On My List</h1>
+            </Jumbotron>
+            {this.state.articles.length ? (
```

Comment:
I think it would make your code more readable if you destructured any state or props objects you are using.